tests/README.md

```diff
@@ -0,0 +1,40 @@
+      Автотесты для тестирования добавления в Избранное на сайте https://prom.ua/
+1. Подготовка среды:  в Pycharm через терминал проинсталлировать зависимости: pip install -r requirements.txt
+2. Запуск в терминале Pycharm: "venv\Scripts\pytest.exe -m ui" -тесты IU
+                               "venv\Scripts\pytest.exe -m rest" -тесты по rest api
```

Comment:
please mark the commands as a code snippets (bordering both sides with ` character), so it would become easier to find them like:

> Подготовка среды: в Pycharm через терминал проинсталлировать зависимости: 

`pip install -r requirements.txt`

Reply:
done (не дон! :))) )